CHANGELOG.md

```
@@ -16,6 +16,10 @@ and this project adheres to
 
 - 🐛(docs) run migration 0027 without superuser role
 
+### Added
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)